placement/service/service.go

```diff
@@ -258,31 +233,5 @@ func (ps placementService) MarkInstanceAvailable(instanceID string) error {
 		return err
 	}
 
-	return ps.s.CheckAndSet(p, v)
-}
-
-func (ps placementService) Placement() (placement.Placement, int, error) {
```

Comment:
We no longer need these in the service interface?

Reply:
We don't, as it's exposed through the embedded Storage interface